consensus/diffs.go

```diff
@@ -174,39 +127,27 @@ func (s *State) applyDiffSet(bn *blockNode, applied bool) {
 		}
 	}
 
-	// The sets of diffs can be applied and reversed in any order, so the
-	// ordering is kept when applying and reversing sets. However, within the
-	// sets, the diffs must be reversed in the opposite order that they were
-	// applied, due to transactions being able to depend on each other. This
-	// results is messier for loops when applied is false.
-	if applied {
-		for _, scod := range bn.siacoinOutputDiffs {
-			s.commitSiacoinOutputDiff(scod, applied)
-		}
-		for _, fcd := range bn.fileContractDiffs {
-			s.commitFileContractDiff(fcd, applied)
-		}
-		for _, sfod := range bn.siafundOutputDiffs {
-			s.commitSiafundOutputDiff(sfod, applied)
-		}
+	// apply each of the diffs
```

Comment:
comment style mismatch. All of my comments are approx. complete sentences.

// Like this.